### ISSUE TITLE
Extended API for this.off

### ISSUE THREAD

**Issue by hv-pul** [hv-pul/hv-boilerpate-rfcs#4]
# Summary

Using `this.off` removes all event listeners that where added within the module.
It would be nice if it would support the same args as `this.on` to remove specific handlers.

# Basic example

```javascript
this.off('click', '.button');
this.off('click', this.onClick);
this.off(window, 'resize');
this.off(document, 'popstate', this.onPopstate);
```

# Motivation

If we need a mix of lasting and temporary event listeners, we currently need to fall back to regular `addEventListener`/`removeEventListener`. This is inconvenient – there is a reason we have `this.on` with its better API – and adds mental overhead (when to use which).

# Unresolved questions

Not sure if I'm missing something, perhaps this is already possible somehow?
For the implementation, it might be possible to use temporary namespaces?

**Comment by betabong** [comment]
I agree on this and I think that it should be doable without too much effort. Already started improvements in Edelweiss, and I plan on bringing them back to the Boilerplate later on. This would be a good addition though.
